make_UPD_igv.py

```diff
@@ -0,0 +1,176 @@
+#! /usr/bin/env python3
+import sys
+import argparse
+import vcf
+
+def parse_ped(ped_file):
+    samples = {}  # 'sample_id': {'family': 'fam_id', 'parents': ['sample_id', 'sample_id']}
+    for line in ped_file:
```

Comment:
you have not opened the ped_file yet with `with open(ped_file, 'r') as ped_input_file: `?

Reply:
Ped file is opened in argparse: 

parser.add_argument('ped_file', type=argparse.FileType('r'), help='PED file')